tests/cypress/support/commands.js

```diff
@@ -1 +1,13 @@
 import "@testing-library/cypress/add-commands";
+
+Cypress.Commands.add("acceptCookiePolicy", () => {
```

Comment:
I added [this](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/tests/cypress/forms_spec.js#L28) to my test but now I'm wondering if this way is better as it's more user centric?

Reply:
I'd say so.

The only reason why I'd opt for your approach would be speed, but we don't have that much to gain here.

It's important to test the cookie banner in all contexts I think. You can sometimes get bugs with z-index etc. that e2e can catch provided we're actually testing the cookie banner, not skipping it entirely.

Reply:
Makes sense, I'll keep that in mind for next time - thanks for clarifying